src/server/common/helpers/session-cache/utils.js

```diff
@@ -127,3 +152,19 @@ export const updateExemptionSiteDetailsBatch = (
 
   return updatedSiteDetails
 }
+
+export const clearSavedSiteDetails = async (request, h) => {
+  request.yar.clear(SAVED_SITE_DETAILS_CACHE_KEY)
+
+  await request.yar.commit(h)
+}
+
+export const setSavedSiteDetails = async (request, h, value) => {
+  const cacheValue = value || {}
+
+  request.yar.set(SAVED_SITE_DETAILS_CACHE_KEY, cacheValue)
+
+  await request.yar.commit(h)
+
+  return cacheValue
```

Comment:
Could potentially grab the value out of the cache with `yar.get`? It would guarantee the cache has been updated properly?